### ISSUE TITLE
[Tflite Model]The tflite model inception_resnet_v2.tflite(2018.04.27) lacks softmax operation

### ISSUE THREAD

**Issue by Wenzhao-Xiang** [tensorflow/tensorflow#23036]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Linux Ubuntu 16.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:
- **Python version**:
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
I use [intel/webml-polyfill](https://github.com/intel/webml-polyfill) to parse [this model](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/models.md#image-classification-float-models) and do inference, but get the strange output result as the following:

order | Label | Probability
-- | -- | --
1 | bee eater | 1093.54%
2 | jay | 369.31%
3 | brambling | 363.70%

After add a softmax layer, the output result seems more normal:

order | Label | Probability
-- | -- | --
1 | bee eater | 96.99%
2 | jay | 0.07%
3 | brambling | 0.07%

So if the model need another "softmax" operation after "fullyconnect" for the image classification task? 

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by ymodak** [comment]
In neural networks the last layer outputs the final activation i.e final result of our model. Therefore its meaningful to interpret this result as a probability number ranging from (0-1).This squashing of final activation into desired probability value is performed by the softmax activation function of output layer. Therefore it makes sense to add a softmax layer as an output layer to your neural network model.

**Comment by ymodak** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
